### ISSUE TITLE
Error: <tspan> attribute x: Expected length, "-8-0.25em".

### ISSUE THREAD

**Issue by jcdavison** [hshoff/vx#286]
hi @hshoff 

Cool library, very promising for us. 

We are attempting to render something that plots linear money on the y axis and time on the x axis
```
          <AxisLeft
            top={margin.top}
            left={margin.left}
            labelOffset={11}
            tickLabelProps={(value, index) => ({
              dx: "",
              fill: "black",
              fontSize: 12,
            })}
            scale={yScale}
            label={'Price'}
          />
// axisLeft produces correct tspan elements
// <tspan x="-8" dy="0em">$0</tspan>

          <AxisLeft
            top={margin.top}
            left={margin.left}
            labelOffset={11}
            scale={yScale}
            label={'Price'}
          />

// axisLeft without defined tickLabelProps produces bad tspan elements
// <tspan x="-8-0.25em" dy="0em">0</tspan>

          <AxisBottom
            top={height - margin.bottom}
            left={margin.left}
            scale={xScale}
            label={'Date'} />
```

Our problem occurs when we remove the `tickLabelProps: () => {}` from the `AxisLeft` component.

The graph renders correctly but we keep getting html errors where the computed value of the tspan is wrong.

When we include the tickLabelProps callback the error goes away.  We are following your docs and are a bit confused by this behavior, please advise a suitable direction or course of action we might take.

**Comment by hshoff** [comment]
Hi @jcdavison thanks for checking out vx! To get him reproduce the bug, which version of `@vx/axis` are you using? The latest version is `0.0.162` and has some fixes for tick labels `<text />` elements

**Comment by lisza** [comment]
Great thank you that fixed it! We were using `@vx/axis@0.0.160` @hshoff
